src/toil_lib/urls.py

```diff
@@ -23,14 +23,15 @@ def download_url(job, url, work_dir='.', name=None, s3_key_path=None, cghub_key_
     :param str name: Name of output file, if None, basename of URL is used
     :param str s3_key_path: Path to 32-byte encryption key if url points to S3 file that uses SSE-C
     :param str cghub_key_path: Path to cghub key used to download from CGHub.
+    :param bool mock: Whether to run docker calls with mock_mode. See :func:`toil_lib.programs.docker_call`
```

Comment:
Yes, that's correct. I misunderstood what you were asking for. I will change the wording.